devops/makeAPIGateway/makeAPIGateway.yml

```diff
@@ -0,0 +1,77 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: AWS API Gateway with a Mock Integration
+
+Resources:
+
+  ApiGatewayRestApi:
+    Type: AWS::ApiGateway::RestApi
+    Properties:
+      ApiKeySourceType: HEADER
+      Description: An API Gateway with a Mock Integration
+      EndpointConfiguration:
+        Types:
+          - REGIONAL
+      Name: mock-api
```

Comment:
Just checking, is it intentional that the API name is always the same? 

The script shows you what the API name would be using our naming convention, but no name ref is passed to this template in order to use it

<img width="717" alt="image" src="https://user-images.githubusercontent.com/30481844/154184190-a6759302-6f49-4ece-beb0-f67775766ca9.png">

Reply:
I made the names unique now to cause less confusion